content/docs/quickstart/aws/destroy-stack.md

```diff
@@ -42,7 +42,9 @@ Resources:
 Duration: 26s
 ```
 
-To delete the stack itself, run `pulumi stack rm`.
+To delete the stack itself, run `pulumi stack rm`. Note that this removes the stack
+entirely from `pulumi.com`, along with all of its update history. See [pulumi stack
```

Comment:
Instead of the "See pulumi stack rm for more details", could we just add a link around the `pulumi stack rm` in the first sentence?

The link should use `relref`.

Feedback applies to this page for other clouds as well.